tfrecord/reader.py

```diff
@@ -69,14 +69,14 @@ def read_records(start_offset=None, end_offset=None):
     file.close()
 
 
-def tfrecord_loader(data_path, index_path, description, shard=None):
-    """Create an iterator from a tfrecord dataset. 
+def tfrecord_loader(data_path, index_path, description=None, shard=None):
+    """Create an iterator from a tfrecord dataset.
 
     Args:
         data_path: Path of the input data.
         index_path: Path of index file. This can be set to None if not available.
-        description: A dictionary of key and values where keys are the name of the features and values correspond to
-                     data type. The data type can be "byte", "float" or "int".
+        description: List of keys to extract from each record. If None, then all
+            features contained in the file are extracted. (default: None)
```

Comment:
Should we have add a `DeprecationWarning` now that description is no longer of type dict?